src/mutations.js

```diff
@@ -3,15 +3,27 @@ import {
   FETCH_LIBRARY,
   SET_PASSAGE,
   SET_DISPLAY_MODE_INTERLINEAR,
+  SET_DISPLAY_MODE_NAMED_ENTITIES,
   SET_DISPLAY_MODE_DEFAULT,
+  SELECT_NAMED_ENTITIES,
+  CLEAR_NAMED_ENTITIES,
 } from '@/constants';
 
 export default {
+  [SELECT_NAMED_ENTITIES]: (state, entities) => {
+    state.selectedNamedEntities = entities;
+  },
+  [CLEAR_NAMED_ENTITIES]: state => {
+    state.selectedNamedEntities = [];
+  },
+  [SET_DISPLAY_MODE_NAMED_ENTITIES]: state => {
+    state.displayMode = 'named-entities';
+  },
   [SET_DISPLAY_MODE_INTERLINEAR]: state => {
-    state.interlinear = true;
+    state.displayMode = 'interlinear';
```

Comment:
not a blocker for merge, but as we continue to add more display modes, we might eventually want the equivalent of a Django `ChoiceField` (constants and human-readable label)